bsi_zoo/metrics.py

```diff
@@ -68,7 +57,62 @@ def euclidean_distance(x, x_hat, *args, **kwargs):
             x_hat[active_set_hat], active_set_hat, fwd, tmin=1, tstep=1
         )  # estimate
 
-        # euclidean distance check
+    return stc, stc_hat, fwd
+
+
+def jaccard_error(x, x_hat, *args, **kwargs):
+    # You can read more on the Jaccard score in Scikit-learn definition https://scikit-learn.org/stable/modules/generated/sklearn.metrics.jaccard_score.html
+    return 1 - jaccard_score(x, x_hat, average="samples")
+
+
+def mse(x, x_hat, *args, **kwargs):
+    return mean_squared_error(x, x_hat)
+
+
+def EMD(x, x_hat, *args, **kwargs):
```

Comment:
if the other function is called mse call this one emd
```suggestion
def emd(x, x_hat, *args, **kwargs):
```

and actually it could be:

```suggestion
def emd(x, x_hat, *, orientation_type, subject, nnz, *args, **kwargs):
```

to be sure orientation_type, subject and nnz are passed